src/language/pythonLanguage.ts

```diff
@@ -1,30 +1,67 @@
 import { StringOrRegExp } from '@cucumber/cucumber-expressions'
+import { RegExps } from '@cucumber/cucumber-expressions/dist/cjs/src/ParameterType'
```

Comment:
Please remove unused imports. Also, when you do import, never import from the `dist` directory.